pkg/registry/memory/nse_memory_server.go

```diff
@@ -32,6 +32,15 @@ type memoryNetworkServeRegistry struct {
 }
 
 func (m *memoryNetworkServeRegistry) RegisterNSE(ctx context.Context, registration *registry.NSERegistration) (*registry.NSERegistration, error) {
+	nseRegistration, err := m.processRegistration(registration)
+	if err != nil {
+		return nseRegistration, err
+	}
+
+	return next.NetworkServiceRegistryServer(ctx).RegisterNSE(ctx, registration)
+}
+
+func (m *memoryNetworkServeRegistry) processRegistration(registration *registry.NSERegistration) (*registry.NSERegistration, error) {
```

Comment:
Do we really need this method? Can we in bulk server call `m.RegisterNSE(...)` ?

Reply:
We can't since it will loose stream capability.